app/views/two_factor_authentication/otp_verification/show.html.slim

```diff
@@ -2,14 +2,16 @@
 
 
 h1.heading = t('devise.two_factor_authentication.header_text')
-p.mt-tiny.mb0 == t('instructions.2fa.confirm_code', number: "<strong>#{@phone_number}</strong>")
+p#code-instructs.mt-tiny.mb0 == t('instructions.2fa.confirm_code',
```

Comment:
i think we should standardize pattern to 1element.class1.class2.idname`, so `p#.mt-tiny.mb0#code-instructs` in this case